coderd/files/cache_test.go

```diff
@@ -206,7 +208,8 @@ func TestRelease(t *testing.T) {
 	for closedIdx, id := range ids {
 		stillOpen := len(ids) - closedIdx
 		for closingIdx := range batchSize {
-			c.Release(id)
+			_ = releases[id][0]()
+			releases[id] = releases[id][1:]
```

Comment:
just change the loops to iterate over this `releases` map if you really wanna do it this way

Reply:
I didn't feel like changing the test. This works